### ISSUE TITLE
reword msgbox about context menu

### ISSUE THREAD

**Issue by jeffythedragonslayer** [Skarsnik/QUsb2snes#102]
I suggest rewording this msgbox so that it is clear the context menu is what is being talked about:

![image](https://user-images.githubusercontent.com/13253146/137651052-935d5757-2ac4-4882-9c57-a67c518fc75e.png)

**Comment by Skarsnik** [comment]
I am not sure how to reword that, it's exactly what it does :)

**Comment by jeffythedragonslayer** [comment]
Windows has a lot of menus.  The menu that opens when you right mouse click something and is populated differently depending on where you right clicked is called the context menu:

https://en.wikipedia.org/wiki/Context_menu

**Comment by jeffythedragonslayer** [comment]
How about:

> Do you want to create an entry in the 'Send To' submenu of the Windows right-click context menu to upload files to the sd2snes?
